_pages/tylergarrett.yaml

```diff
@@ -0,0 +1,3 @@
+ githubHandle: tylergarrett
+ pageUrl: tylergarrett.github.io
+ timestamp: 2017-01-25
```

Comment:
@tylergarrett make sure that the syntax matches what's included in the instructions exactly, [here's a link](https://services.github.com/on-demand/github-desktop/13/)